cli/alterations/alteration.py

```diff
@@ -17,52 +17,54 @@
 
 from apis import nifiManagment
 from apis import auxiliaryFunctions
-
+import env
 
 def createMerge(nifiConfiguration,information, name):
-    nameCompose= nameActionReturn(information["action"],name)
+    nameCompose= nameActionReturn(information[env.ALTERATION_ACTION_TAG],name)
     merge=auxiliaryFunctions.prepareforAll("./template/alterations/Merge.json",information)
-    merge = auxiliaryFunctions.addSensitiveVariable(merge, "MergeContent", "Maximum Number of Entries", information["maxMessages"])
+    merge = auxiliaryFunctions.addSensitiveVariable(merge, "MergeContent", "Maximum Number of Entries", information[env.ALTERATION_MAX_MESSAGES_TAG])
     nifiConfiguration.create(nameCompose, merge)
-    nifiConfiguration.changeSchedule(nameCompose, "MergeContent", information["windowSeconds"])
+    nifiConfiguration.changeSchedule(nameCompose, "MergeContent", information[env.ALTERATION_WINDOW_SECONDS_TAG])
 
 
 def createDecode(nifiConfiguration,information,name):
-    nameCompose= nameActionReturn(information["action"],name)
+    nameCompose= nameActionReturn(information[env.ALTERATION_ACTION_TAG],name)
```

Comment:
In general, whenever you access a key of a variable received from an argument, is good idea to check whether the key is present, like:
```
if env.ALTERATION_ACTION_TAG not in information:
    raise ValueError(f"Missing required key: {env.ALTERATION_ACTION_TAG}")
```

Reply:
Code added